script/install.sh

```diff
@@ -192,6 +193,7 @@ Description
 Options
 
   -a --activate     Only print the path to the activate script
+  -w --where        Only print the path to the compiler binary
```

Comment:
If I understand this code correctly, it's not the path to the compiler binary.  It's the path to the directory where the compiler binary has been installed.